Browser.py

```diff
@@ -196,10 +186,21 @@ def checkNewDrops(self, lastCheckTime):
             res = self.client.get("https://account.service.lolesports.com/fandom-account/v1/earnedDrops?locale=en_GB&site=LOLESPORTS", headers=headers)
             resJson = res.json()
             return [drop for drop in resJson if lastCheckTime <= drop["unlockedDateMillis"]]
-        except KeyError:
+        except (KeyError, TypeError):
             self.log.debug("Drop check failed")
             return []
 
+    def __needSessionRefresh(self) -> bool:
+        if "access_token" in self.client.cookies.get_dict():
+            res = jwt.decode(self.client.cookies.get_dict()["access_token"], options={"verify_signature": False})
+            timeLeft = res['exp'] - int(time())
+            self.log.debug(f" s until session expires.")
+            if timeLeft < 600:
+                return True
+            return False
+        else:
+            raise NoAccessTokenException()
```

Comment:
This can help clarify your code by using an early return.

```suggestion
        if "access_token" not in self.client.cookies.get_dict():
            raise NoAccessTokenException()

        res = jwt.decode(self.client.cookies.get_dict()["access_token"], options={"verify_signature": False})
        timeLeft = res['exp'] - int(time())
        self.log.debug(f" s until session expires.")
        if timeLeft < 600:
            return True
        return False
```